### ISSUE TITLE
Roll out GitHubIssue annotation over failing RSE integration tests

### ISSUE THREAD

**Issue by jadecarino** [galasa-dev/projectmanagement#1065]
- Add GitHubIssue (Issue 1060) annotation to failing RSE tests
- Update galasactl to show Failed With Defects result in a) console log b) test outputs (YAML, JSON and JUnit)
- Ensure these don't break: a) Jenkins pipeline b) Slack output

**Comment by jadecarino** [comment]
GitHubIssue has been added to all failing RSE tests. They show as Failed With Defects in the Eclipse run editor and in parts of the galasactl log. 

I have made updates to galasactl to add a Failed With Defects section to the Final report in the console however have hit a blocker, PRs to the CLI repo are not building, needs further investigation. 

Regression pipeline in Jenkins seems to be handling the Failed With Defects results fine, however they do still show as Failures in Slack bot. Unsure if Slack bot can recognise custom results.
